lib/browser.js

```diff
@@ -60,6 +60,8 @@ class Browser {
 
     if (helper.isDefined(info.log)) {
       this.emitter.emit('browser_log', this, info.log, info.type)
+    } else if (helper.isDefined(info.total)) {
+      this.lastResult.total = info.total
```

Comment:
If you guard the update to results with state `EXECUTING` then the tests will pass.  You can add a similar test with mock state `EXECUTING` to verify the change.

Reply:
In order to fix the test `should ignore if browser not executing` I would suggest to wrap all those if-clauses with a check for state `EXECUTING`.
Should `this.refreshNoActivityTimeout()` be still called then?

Reply:
I think we should log events in any state, they could be error messages for corner cases.  So remove the line in the test that checks the `.dump` path.

The  `this.refreshNoActivityTimeout()` should always be called, it signals that the browser is still on the line.

Reply:
Makes sense. I've pushed my changes.